### PR TITLE
Edit Command

### PR DESCRIPTION
**Pull request by underthehai**

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 1]
```diff
@@ -1,35 +1,95 @@
...
     }
 
-    //Parse EditActivity
     @Test
-    public void parse_invalidArgs_throwsParseException() {
-        assertWanderLustParseEditCommandFailure(parser, " -activity i/2",
+    public void parse_editActivity_success() {
+        EditDescriptor editDescriptor = new EditDescriptor();
+        editDescriptor.setName(new Name("Ice Skating"));
+        assertWanderLustParseEditCommandSuccess(parser, " -activity 1 n/Ice Skating",
```
Comment (lyeyixian) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:39]: This method name I changed to assertEditParseSuccess(), wait I merge then you update

**Review by lyeyixian — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java (hunk 1 above)
Comment (underthehai) [reply src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:39]: okok

**Review by jiaweiteo — APPROVED** [review]
LGTM

**Pull request merged**